src/test/groovy/com/cloudbees/groovy/cps/CpsTransformerTest.groovy

```diff
@@ -623,4 +624,24 @@ class CpsTransformerTest extends AbstractGroovyCpsTest {
             return x;
         """) == 6;
     }
+
+    @Issue('https://github.com/cloudbees/groovy-cps/issues/26')
+    @Test
+    void interfaceDeclaration() {
+        assert evalCPS('''
+            interface Strategy {
+                Closure process(Object event)
+            }
+            return true
+        ''') == true;
```

Comment:
Would be more reassuring to verify that you can actually *use* the newly defined interface somehow. Implement it, define a variable of interface type, assign an instance of the implementing class, call the interface method.